### PR TITLE
CCTRI-1168/add only uniq indicators into response

### PR DESCRIPTION
**Pull request by GeorgiyII**

### CODE REVIEW

File: api/enrich.py [hunk 1]
```diff
@@ -267,14 +267,20 @@ def observe_observables():
         breaches = output['results']
         breaches.sort(key=lambda x: x['spycloud_publish_date'], reverse=True)
 
+        uniq_catalog_id_list = []
```
Comment (gdavoian) [inline api/enrich.py:270]: Why `list`? There are data structures that are much better for fast lookups (e.g. `set`).

Comment (gdavoian) [reply api/enrich.py:270]: `uniq` is a Unix tool, but the proper spelling is `unique` :)

Comment (GeorgiyII) [reply api/enrich.py]: done

File: api/enrich.py [hunk 2]
```diff
@@ -267,14 +267,20 @@ def observe_observables():
         breaches = output['results']
         breaches.sort(key=lambda x: x['spycloud_publish_date'], reverse=True)
 
+        uniq_catalog_id_list = set()
```
Comment (gdavoian) [inline api/enrich.py:270]: Much better now, but the variable isn't a list anymore, so `unique_catalog_id_set` will make more sense.

Comment (GeorgiyII) [reply api/enrich.py]: done

**Pull request merged**